docs/user-guide/docs-source/setup.md

```diff
@@ -0,0 +1,123 @@
+
+Once installed, launch the Houdini Launcher.
+
+Click the <number-badge>1</number-badge> "Install" button and select <number-badge>2</number-badge> the latest "Production Build" of Houdini 19.0.x.
+
+![Houdini Launcher annotated](images/setup/HoudiniInstall_1.png)
+
+When prompted with the Preferences dialog keep the default settings and enable the <number-badge>3</number-badge> "Unreal Plug-In" and <number-badge>4</number-badge> "Automatically Install SideFX Labs" options, then click <number-badge>5</number-badge> "Save".
+
+![Houdini Launcher License Tab](images/setup/HoudiniInstall_1.1.png)
+
+Once installation completes click on the <number-badge>6</number-badge> "Licenses" tab. Then click the <number-badge>7</number-badge> "License Administrator" button.
+
+![Houdini Launcher License Tab](images/setup/HoudiniInstall_2.png)
+
+Two windows will appear. The "Entitlements" window should display your available "Houdini Engine for Unity/Unreal" licenses. Set the <number-badge>8</number-badge> "Quantity" value to "1" and click the <number-badge>9</number-badge> "Install" button.
+
+![Houdini Administrator Entitlements](images/setup/HoudiniInstall_3.png)
+
+The remaining Houdini Administrator window should now reflect that you've successfully configured your Houdini license. You may close both this window and the Houdini Launcher.
+
+![Houdini Administrator complete](images/setup/HoudiniInstall_4.png)
+
+Next, you'll add the Houdini Engine plugin to your Unreal Engine install. Do this by copying the "HoudiniEngine" folder from...
+
+`C:\Program Files\Side Effects Software\Houdini Engine\Unreal\19.0.x\4.27\HoudiniEngine`
+
+...to the following location...
+
+`C:\Program Files\Epic Games\UE_4.27\Engine\Plugins\Runtime\HoudiniEngine`
+
+
+
+### 4. Add the AWSAmbit.hda to your Houdini Engine install
+
+The final part of setting up Houdini Engine is to copy one of the Ambit plugin files to your Houdini install folder. Recall that you downloaded the Ambit plugin in step 1. Find this file...
+
+`[AwsAmbit plugin folder]\AWSAmbit.hda`
+
+...and copy it to the following location, creating the "otls" directory if necessary...
+
+`C:\Users\[username]\Documents\houdini19.0\otls\AWSAmbit.hda`
+
+
+
+### 5. Install the glTF Exporter UE4 plugin
+
+Some of Ambit's export functionality relies on the **glTF Exporter plugin** from Epic Games. This plugin doesn't ship with Unreal Engine but can be downloaded for free from the Unreal Engine Marketplace following the steps below.
+
+Visit the [glTF Exporter Marketplace web page](https://www.unrealengine.com/marketplace/en-US/product/gltf-exporter).
+
+If not already signed in, click the "Sign in to Buy" button and log in.
+
+Once logged in, click the "Open in Launcher" button. This will take you to the same asset page in the Marketplace tab of the Epic Games Launcher.
+
+Click the "Install to Engine" button and complete the installation.
+
+Congratulations! Initial setup is now complete. 🎉
+
+
+
+## Per Project Setup
```

Comment:
If this is a general user setup (which it is written like it is), then we may want to express that this is only per project if it is not installed to the Engine directory. In this case, Ambit only needs to be installed once.

Reply:
Also, installing to engine directory is _not_ the preferred method unless the user intends to build the entire engine from source. This should be made note of if we are making a mention of it.

Reply:
I mentioned this in another comment, but the docs are now more opinionated and no longer give users multiple options for where to install things. We no longer give the user the option of installing the Ambit plugin at the engine level. I'm happy to share my reasoning for this decision offline if you think we should reconsider.

Reply:
I'm okay with not mentioning the option of installing at the engine level. We don't want to end up debugging users issues if they choose to build the engine from source.